cmd/mb-main.go

```diff
@@ -29,7 +29,7 @@ var (
 	mbFlags = []cli.Flag{
 		cli.StringFlag{
 			Name:  "region",
-			Value: "us-east-1",
+			Value: "",
```

Comment:
Using empty string deliberately here - minio-go overrides region specified in client constructor only if location string is empty.  If  mb call passes us-east-1 as region, all mc mb calls without region will fail with header mismatch and confusing error message that the location should not be us-east-1.